setup.cfg

```diff
@@ -18,6 +18,8 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```

Comment:
Since 3.11 tests weren't passing, you should probably remove this.

Reply:
Done in 11f3224